double-conversion/double-to-string.h

```diff
@@ -111,7 +111,8 @@ class DoubleToStringConverter {
                           int decimal_in_shortest_low,
                           int decimal_in_shortest_high,
                           int max_leading_padding_zeroes_in_precision_mode,
-                          int max_trailing_padding_zeroes_in_precision_mode)
+                          int max_trailing_padding_zeroes_in_precision_mode,
+                          int min_exponent_width = 0)
```

Comment:
Also add a line (or two) to the comment of the constructor, explaining what the parameter does.
Shouldn't need a lot of explanation (quite self-explanatory), but mention that the exponent will never have more than 5 digits.
Then also add a comment to the constant (`5`) in double-to-string.cc and state that changing this constant also requires updating the comment in double-to-string.h

Reply:
Thanks for your advice, I added some comments to introduce this option